diagnostic_aggregator/src/aggregator.cpp

```diff
@@ -310,9 +310,12 @@ void Aggregator::publishData()
 
   agg_pub_.publish(diag_array);
 
-  // Top level is error if we have stale items, unless all stale
```

Comment:
This sounds like a design choice.

Would you mind opening a similar PR on the official repo? Or an issue.
Perhaps for the ROS2 as ROS1 seems unmaintained.